cinema/admin.py

```diff
@@ -0,0 +1,3 @@
+from django.contrib import admin
+
+# Register your models here.

```

Comment:
You need to register your models here to make them manageable through the Django admin interface. Use `admin.site.register(YourModel)` for each model you want to register.